autopts/ptsprojects/zephyr/pbp.py

```diff
@@ -0,0 +1,87 @@
+#
+# auto-pts - The Bluetooth PTS Automation Framework
+#
+# Copyright (c) 2025, Nordic Semiconductor ASA.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms and conditions of the GNU General Public License,
+# version 2, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+import struct
+
+from autopts.pybtp import btp
+from autopts.client import get_unique_name
+from autopts.ptsprojects.stack import get_stack
+from autopts.ptsprojects.testcase import TestFunc
+from autopts.ptsprojects.zephyr.pbp_wid import pbp_wid_hdl
+from autopts.ptsprojects.zephyr.ztestcase import ZTestCase
+from autopts.pybtp.types import Addr
+from autopts.utils import ResultWithFlag
+
+broadcast_code = '0102680553F1415AA265BBAFC6EA03B8'
```

Comment:
Since this is done almost everywhere else, it is fine for now, but generally we should avoid global variables, and if we do want to use global variables, it is common to use UPPER_CASE for global variables so they are no confused with local variables

Reply:
Changed to uppercase